mailbox/lucene/src/test/java/org/apache/james/mailbox/lucene/search/LuceneMemorySearchHighLighterTest.java

```diff
@@ -112,4 +126,46 @@ public void applyRightsCommand(MailboxId mailboxId, Username owner, Username del
             MailboxACL.command().forUser(delegated).rights(MailboxACL.FULL_RIGHTS).asAddition(),
             session(owner))).run();
     }
+
+
+    @Test
+    void shouldHighlightAttachmentTextContentWhenTextBodyDoesNotMatch() throws Exception {
```

Comment:
Any reason why it is not in the contract?

Reply:
moved it to contract class